pkg/options/release.go

```diff
@@ -0,0 +1,53 @@
+package options
```

Comment:
Should this also handle snapshot options?  This is currently very specific to server options rather than a generic package.

Reply:
Resolved. Different use cases require different solutions.  Keep release options as a package and snapshot options as an options.go in main.